test/scripts/build-common.sh

```diff
@@ -0,0 +1,29 @@
+# Lets make sure the build folder was cleared out correctly
```

Comment:
I think you want a "-e" in somewhere so that the script exists immediately on error.  I now tend just to use this at the top of any scripts I write:
```
#!/bin/bash -e
```